templates/flat-black.php

```diff
@@ -7,20 +7,20 @@
 
 		<div id="list">
 			<?php
-			$total = count( $songs );
-			for ( $i = 0; $i < $total; $i++ ) {
+		
+			foreach($songs as $key => $song ) {
```

Comment:
I tend to follow WP coding standards. So, can you please add a space before `$songs`?